src/Angor/Shared/WalletOperations.cs

```diff
@@ -128,10 +128,92 @@ public string GenerateWalletWords()
             .SendEstimatedFees(new FeeRate(Money.Coins(sendInfo.FeeRate)));
 
         var signedTransaction = builder.BuildTransaction(true);
-        
+
         return await PublishTransactionAsync(network, signedTransaction);
     }
 
+    private void UpdatePendingLists(AccountInfo accountInfo)
+    {
+        // remove from the pending remove list if it was removed from the indexer
+        var pendingRemove = accountInfo.PendingRemove.ToList();
```

Comment:
Perhaps a name like pending Spent confirmation and pending receive confirmation make it easier to understand what these are